Makefile

```diff
@@ -0,0 +1,23 @@
+install:
+	pipenv install --dev
+
+package_vulnerability:
+	PIPENV_PYUP_API_KEY="" pipenv check
+
+flake:
+	pipenv run flake8 .
+
+vulture:
+	pipenv run vulture
```

Comment:
```suggestion
	pipenv run vulture .
```

Reply:
good idea, I'll check it still works - has highlighted some 'DB_ACTION_CERTIFICATES = os.getenv('DB_ACTION_CERTIFICATES', '')'  in Config.py

Reply:
Changed the pyproject.toml to point at "." rather than "acceptance_tests".